Dockerfile

```diff
@@ -67,6 +72,7 @@ RUN apt-get update && apt-get install -y \
 	xfsprogs \
 	libzfs-dev \
 	tar \
+	linux-image-`uname -r` \
```

Comment:
Just to make sure I understand the comments above: the `Dockerfile` being commented on here is in some sense an "internal detail" of the way CI and most docker engine developers build/test the codebase. The set of content that an "at runtime" installation of Docker + the C/R feature on various supported and unsupported Linux platforms requires should be handled in the experimental docs for this feature. This Dockerfile starts with `FROM ubuntu:14.04` and so I think that the expectation is the `RUN` command installing pre-reqs is strongly tied to that distro's package set, both in name and content.

This does not mean we don't need good documentation on how to set up a runtime system with all the right components, but given the target here is experimental for the meantime, I think we are satisfied that the docs coverage of installing criu, either via package or from source is sufficient for experimental.

Am I missing anything that we need to deal with here in the build-time `Dockerfile` regarding other hosts?